uxarray/grid/grid.py

```diff
@@ -232,6 +234,52 @@ def from_dataset(
 
         return cls(grid_ds, source_grid_spec, source_dims_dict)
 
+    @classmethod
+    def from_file(
+        cls,
+        filename: str,
+        backend: Optional[str] = "geopandas",
+        **kwargs,
+    ):
+        """Constructs a ``Grid`` object from a using geopandas read_file.
```

Comment:
This should now reflect the new changes.

""Attempts to read a grid file using either Xarray or GeoPandas and constructs a ``Grid`` object.